README.md

```diff
@@ -129,7 +129,7 @@ const MyComponent = () => {
 
 #### Code-loading
 
-Deliver a light, interactive core experience to users and progressively add high-end-only features on top, if a users hardware can handle it. Below is an example using the Network Status hook:
+Deliver a light, interactive core experience to users and progressively add high-end-only features on top, if a user's hardware can handle it. Below is an example using the Network Status hook:
```

Comment:
I don't think this should have an apostrophe added here as it's a plural

Reply:
`if a users hardware`, must be changed to `if users' hardware` or `if a user's`